### PR TITLE
boards: nucleo_wl55jc: assert SRST before any target connection

### PR DESCRIPTION
**Pull request by jelledevleeschouwer**
Failed to flash blinky example to stock nucleo_wl55jc due to default
application entering low power mode using WFI. To connect to a sleeping
stm32wl55 a system reset is required.

### CODE REVIEW

**Comment by jelledevleeschouwer** [comment]
Hi @erwango,

Looks like some compliance check workflow runs are waiting for your approval. Could you maybe have a look at this please?

Thanks,
Beste Regards,
Jelle

**Review by erwango — APPROVED** [review]

**Review by FRASTM — APPROVED** [review]

**Comment by jelledevleeschouwer** [comment]
Thanks for reviewing, @erwango & @FRASTM. I am not authorized to merge, maybe someone else is willing and authorized to do so? Possibly @ABOSTM? Thanks,

Regards,
Jelle

**Comment by erwango** [comment]
> Thanks for reviewing, @erwango & @FRASTM. I am not authorized to merge, maybe someone else is willing and authorized to do so? Possibly @ABOSTM? Thanks,
> 
> Regards, Jelle

Only people from the release engineering team have merge rights. I've just added 2 of them.

**Review by ABOSTM — APPROVED** [review]

**Pull request merged**
